client/verta/tests/conftest.py

```diff
@@ -251,6 +256,8 @@ def dir_and_files(strs, tmp_path):
     filepaths : set of str
 
     """
+    dirpath = tmp_path / request.param
```

Comment:
Does `string / string` actually work!? Python went too far!

Reply:
The pytest-provided `tmp_path` [is actually a `pathlib.Path` object](https://docs.pytest.org/en/stable/reference.html#std-fixture-tmp_path) rather than a string, which [uses division for joining paths](https://docs.python.org/3/library/pathlib.html#operators)